tests/tests_run.sh

```diff
@@ -7,7 +7,8 @@ export PYTHONFAULTHANDLER=1
 
 bin/neuralmonkey-train tests/vocab.ini
 bin/neuralmonkey-train tests/bahdanau.ini
-bin/neuralmonkey-train tests/bpe.ini
+NEURALMONKEY_STRICT= bin/neuralmonkey-train tests/bpe.ini
+bin/neuralmonkey-train tests/bpe.ini -s 'decoder.encoders=[<encoder_frozen>]'
```

Comment:
teďkon to nainicializuje enkodér na čisto a nic to s tim nedělá? Nechceme sem ještě přidat `-s 'main.initial_variables=["soubor co by měl existovat z nějakýho z předešlých běhů"]`?